src/mscorlib/src/System/String.cs

```diff
@@ -49,7 +49,7 @@ public sealed class String : IComparable, ICloneable, IConvertible, IEnumerable
 
         // For empty strings, this will be '\0' since
         // strings are both null-terminated and length prefixed
-        [NonSerialized] private char m_firstChar;
+        [NonSerialized] internal char m_firstChar;
```

Comment:
@jkotas, now that `EditorBrowsableAttribute` is in corelib maybe we could mark this with `EditorBrowsableState.Never` and use it directly from places such as `char` / `StringBuilder`?